corehq/apps/reports/standard/sms.py

```diff
@@ -707,6 +708,12 @@ def get_queryset(self):
                 Q(messagingsubevent__content_type__in=content_type_filter)),
         )
 
+        event_status = EventStatusFilter.get_value(self.request, self.domain)
+        if event_status is not None:
+            data = data.filter(
```

Comment:
it doesn't seem like it's that much work to add the 3 conditional filters.  maybe we could save the errors to the subevent and event when they happen to make this easier going forward, but i think doing it the way i described is the easiest way to add this filter now